Test/comp/Libraries/consumer.dfy

```diff
@@ -1,9 +1,9 @@
 // RUN: %dafny /library:"%S/Inputs/directLibrary.dfy" /library:"%S/Inputs/secondLibrary.dfy" /compile:0 /spillTargetCode:3 "%s" > "%t"
 // RUN: %diff "%s.expect" "%t"
 // RUN: %OutputCheck "%s" --file-to-check="%S/consumer.cs"
+// CHECK: GloballyUniqueProducer
```

Comment:
Maybe with different names like "DirectLibraryProducer", "IndirectLibraryProducer" and "NonLibraryProducer" it could be easier to grok the test.